### PR TITLE
Use the "reply-to" header of the original post when answering via mail

### PR DESCRIPTION
**Pull request by annando**
This is continuing the work from PR https://github.com/friendica/friendica/pull/7851

Additionally some small bug was fixed: the delivery counter hadn't been increased for mail deliveries.

### CODE REVIEW

File: src/Worker/Delivery.php [hunk 1]
```diff
@@ -481,10 +481,11 @@ private static function deliverDiaspora($cmd, $contact, $owner, $items, $target_
 	 * @param array  $contact     Contact record of the receiver
 	 * @param array  $owner       Owner record of the sender
 	 * @param array  $target_item Item record of the content
+	 * @param array  $thr_parent  Item record of the thread parent
```
Comment (MrPetovan) [inline src/Worker/Delivery.php]: `parent thread`

File: src/Worker/Delivery.php [hunk 2]
```diff
@@ -562,5 +574,9 @@ private static function deliverMail($cmd, $contact, $owner, $target_item)
 		}
 
 		Email::send($addr, $subject, $headers, $target_item);
+
+		Model\ItemDeliveryData::incrementQueueFailed($target_item['id']);
```
Comment (MrPetovan) [inline src/Worker/Delivery.php]: Why queue failed?

Comment (annando) [reply src/Worker/Delivery.php]: Because of "copy and paste" without thinking :-)

Comment (MrPetovan) [reply src/Worker/Delivery.php]: Tsk tsk tsk...

**Pull request merged**